docs/user-guide/pods/init-container.md

```diff
@@ -0,0 +1,168 @@
+---
+assignees:
+- erictune
+
+---
+
+* TOC
+{:toc}
+
+In addition to having one or more main containers (or **app containers**), a
+pod can also have one or more **init containers** which run before the app
+containers.   Init containers allow you to reduce and reorganize setup scripts
+and "glue code".
+
+## Summary
+
+An init container is exactly like a regular container, except that it always
+runs to completion and each init container must complete successfully before
+the next one is started. If the init container fails (exits with a non-zero
+exit code) on a `RestartNever` pod the pod will fail - otherwise it will be
+restarted until it succeeds or the user deletes the pod.
+
+Init containers are currently specified by adding an annotation.  The
```

Comment:
"Currently"? Will this change?

How about "You specify a container as an init container by adding an annotation."